frontend/src/components/sessions/results/StudentList.tsx

```diff
@@ -0,0 +1,128 @@
+import React, { useMemo, useState } from "react";
+import {
+  Box,
+  Button,
+  Divider,
+  Flex,
+  ListItem,
+  OrderedList,
+  Spacer,
+  VStack,
+} from "@chakra-ui/react";
+
+import type { StudentResponse } from "../../../APIClients/types/ClassClientTypes";
+import { includesIgnoreCase } from "../../../utils/GeneralUtils";
+import SearchBar from "../../common/table/SearchBar";
+import SortMenu from "../../common/table/SortMenu";
+
+type StudentListItemProps = {
+  firstName: string;
+  lastName: string;
+  isViewed: boolean;
+  isSelected: boolean;
+  onClick: () => void;
+};
+
+const StudentListItem = ({
```

Comment:
why do we keep this in the same component? vs. just separating out into its own?

Reply:
I didn't think the item component would be used outside this file, and I was hesitant to add an extra file since I can see the number of files in this particular directory getting quite large. If you have an idea for how I can organize the files such that making a separate file is clean, I'm more than happy to make the change!

Reply:
i'm okay with it either way! was just curious.

although, in general, i don't think we should worry about # of files? i feel like we make sub-directories quite liberally & reorganize our folder structure frequently, so definitely feel free to do so as well! (we're definitely overdue for some cleaning 🧹). afaik, no drawbacks to that? let me know, if i'm wrong haha.

also, if the directory is not yet large, instead of preemptively trying to create less files, i feel like we can just organize when we get to that point. just my 2 cents!

Reply:
I did it this way because I'm working on other components in // and it was getting kinda large lol... But yes, fair enough. I'll make a note of this for myself and come back to it in a separate PR once I have more components to organize.